lib/dispatch-rider/demultiplexer.rb

```diff
@@ -18,6 +18,7 @@ def initialize(queue, dispatcher, error_handler)
     def start
       do_loop do
         begin
+          sleep 1
```

Comment:
Why one second? Maybe `sleep(0.5)` or even `sleep(0.25)` is enough and throttles it less?

Reply:
because once every seconds is enough